aio_pika/queue.py

```diff
@@ -484,6 +484,11 @@ def __init__(self, queue: Queue, **kwargs: Any):
         self._queue = asyncio.Queue()
         self._consume_kwargs = kwargs
 
+        if kwargs.get("timeout"):
```

Comment:
Hm... that's should be handled by wait_for implementation: https://github.com/python/cpython/blob/main/Lib/asyncio/tasks.py#L494-L507

Reply:
yep... and `asyncio.wait_for` calls: 
https://github.com/python/cpython/blob/main/Lib/asyncio/timeouts.py#L145-L162 and so on which gives ~5% overhead even if we don't use `timeout=` argument.
my idea is to eliminate this calls' chain if we don't need it